tests/cnf/core/network/metallb/internal/frr/frr.go

```diff
@@ -230,6 +269,60 @@ func DefineBGPConfigWithStaticRouteAndNetwork(localBGPASN, remoteBGPASN int, hub
 	return bgpConfig
 }
 
+// DefineBGPConfigWithUnnumbered defines BGP config file with static route and network.
+func DefineBGPConfigWithUnnumbered(localBGPASN, remoteBGPASN int, interfaceName, peerLinkLocalAddress string,
```

Comment:
Who is doing the RAs?

Reply:
No, I mean who is sending out Router Adv in the ipv6 protocol on that L2?

Reply:
Which of the two is initiating the TCP connection? 
On the frrk8pod side it is using for two test cases dynamicAS. The external side is always configured with a static AS.

Reply:
Had a look at the FRR config and dont see an option for it passivly listen to RA notifications. So both sides are trying.